src/app/services/sms/sms_count.server.model.ts

```diff
@@ -34,6 +34,9 @@ const VerificationSmsCountSchema = new Schema<IVerificationSmsCountSchema>({
       required: true,
```

Comment:
this PR should be purely the script and not contain anything else.

I think adding a new schema key (and possibly the programatic setting of whether `isOnboardedAccount`) should be done in a separate PR.

Reply:
agreed, will remove this and add it in the follow up PR